### PR TITLE
Searching - Display results

### PR DESCRIPTION
**Pull request by Matthew5002**
Added business and search views

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client'
 import MapView from '@components/MapView'
+//import SearchView from '@components/SearchView'
+//import BusinessView from '@components/BusinessView'
```
Comment (bridges-wood) [inline src/App.tsx:4]: Remove commented imports.

File: src/components/Business.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type Business = {
+    id: any;
+    businessName: string;
+    distance: string;
+    futureProofRating: Number;
+    userRating: Number;
+    logoURL: string
+  };
+
+  export default Business
```
Comment (bridges-wood) [inline src/components/Business.ts:10]: You can import this from the typings directory using `import {Business} from '@typings/types'`

File: src/components/BusinessView.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+const ExampleBusiness: Business = {
+  id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28ba',
+  businessName: 'Starbucks Coffee',
+  distance: '1.2',
+  futureProofRating: 75,
+  userRating: 62,
+  logoURL: '../../assets/icon.png'
+}
+
+const ExampleBusinessDescription = "Starbucks Corporation is an American multinational chain of coffeehouses and roastery reserves headquartered in Seattle, Washington. It is the world's largest coffeehouse chain."
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:14]: Make sure that this gets removed when the resolvers are properly set up. 👍🏻

File: src/components/BusinessView.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
+import React, {useState} from "react"
+import { SafeAreaView, View, FlatList, Text, Image, StyleSheet, TouchableOpacity, ScrollView } from "react-native"
+import Business from './Business'
+
+const ExampleBusiness: Business = {
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:5]: You'll be able to either create a more refined definition using the `Pick` or `Omit` [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) and the imported Business type from `@typings/types`.

File: src/components/BusinessView.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  futureProofRating: 75,
+  userRating: 62,
+  logoURL: '../../assets/icon.png'
+}
+
+const ExampleBusinessDescription = "Starbucks Corporation is an American multinational chain of coffeehouses and roastery reserves headquartered in Seattle, Washington. It is the world's largest coffeehouse chain."
+
+const TitleView = ({business}: {business: Business}) => (
+  <View style={{alignItems: "center"}}>
+    <Image source={require('../../assets/icon.png')} style={{width: 90, height: 90, borderRadius: 10}} resizeMode="contain"/>
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:18]: Can you not get this from `ExampleBusiness`?

File: src/components/BusinessView.tsx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    <Text style={styles.titleText}>{business.businessName}</Text>
+    <Text style={styles.subtitleText}>{"2 Bath Street, Bath, BA1 1AA"}</Text>
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:20]: Why is this not defined using data from the input object?

File: src/components/BusinessView.tsx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+  businessName: 'Starbucks Coffee',
+  distance: '1.2',
+  futureProofRating: 75,
+  userRating: 62,
+  logoURL: '../../assets/icon.png'
+}
+
+const ExampleBusinessDescription = "Starbucks Corporation is an American multinational chain of coffeehouses and roastery reserves headquartered in Seattle, Washington. It is the world's largest coffeehouse chain."
+
+const TitleView = ({business}: {business: Business}) => (
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:16]: It would be good to refactor this to its own file and import it. I understand the logic of wanting it to be close to the stuff, but I think the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) will serve us better long term.

File: src/components/BusinessView.tsx [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+  <View>
+    <Text style={styles.headingText}>{"DESCRIPTION"}</Text>
+    <View style={styles.itemBackgroundStyle}>
+      <View style={{flexDirection: "row", alignItems: "center", paddingHorizontal: 15}}>
+        <Image source={require('../../assets/icon_paragraph.png')} style={{width: 25, height: 25}} resizeMode="contain"/>
+        <Text style={styles.bodyText}>{descriptionText}</Text>
+      </View>
+    </View>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:34]: Refactor to own file.

File: src/components/BusinessView.tsx [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+const RatingView = ({ratingName, ratingValue}: {ratingName: string, ratingValue: Number}) => (
+  <View style={styles.itemBackgroundStyle}>
+    <View style={{alignItems: "center"}}>
+      <Text>{ratingName + " Rating"}</Text>
+      <Text style={styles.subtitleText}>{"Tap for Details"}</Text>
+    </View>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:43]: Refactor to own file.

File: src/components/BusinessView.tsx [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+const RatingsView = ({futureProofRating, userRating}: {futureProofRating: Number, userRating: Number}) => (
+  <View>
+    <Text style={styles.headingText}>{"RATINGS"}</Text>
+    <View style={{flexDirection: "row", justifyContent: "space-between"}}>
+      <RatingView ratingName="FutureProof" ratingValue={futureProofRating}/>
+      <RatingView ratingName="Consumer" ratingValue={futureProofRating}/>
+    </View>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:53]: Refactor to own file.

File: src/components/BusinessView.tsx [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+const ImagesCarousel = () => (
+  <View>
+    <Text style={styles.headingText}>{"DESCRIPTION"}</Text>
+    <Image source={require('../../assets/Sample_Business_Image.png')} style={{width: 250, height: 300, borderRadius: 10, alignSelf: "center"}} resizeMode="contain"/>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:60]: Refactor to own file.

File: src/components/BusinessView.tsx [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+const BusinessView = () => {
```
Comment (bridges-wood) [inline src/components/BusinessView.tsx:62]: Should this component take a Business object as a parameter?

File: src/components/SearchView.tsx [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+    return "#50A75C"
+  }
+  if (ratingValue > 50) {
+    return "#BED62E"
+  }
+  if (ratingValue > 30) {
+    return "#D6C52E"
+  }
+  return "brown"
+}
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:46]: In terms of style, I'd like to see these redefined as arrow functions with the if statements on a single line, no brackets. Also, it would be good to add some JSDoc.

File: src/components/SearchView.tsx [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+    return "Good"
+  }
+  if (ratingValue > 50) {
+    return "Okay"
+  }
+  if (ratingValue > 30) {
+    return "Bad"
+  }
+  return "Avoid"
+}
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:62]: Same as above.

File: src/components/SearchView.tsx [hunk 15]
```diff
@@ -0,0 +1,163 @@
...
+
+const RatingCapsuleHeight = 18;
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:64]: For constants, it would be great to define them in [Screaming Snake Case](https://en.wikipedia.org/wiki/SCREAMING_SNAKE_CASE) and accompany them with a [JSDoc](https://jsdoc.app/about-getting-started.html) comment to explain them.

File: src/components/SearchView.tsx [hunk 16]
```diff
@@ -0,0 +1,163 @@
...
+
+const RatingCapsule = ({ratingValue}: {ratingValue: Number}) => (
+  <View style={{alignItems: "center", backgroundColor: ratingValToColour(ratingValue), width: 55, height: RatingCapsuleHeight, borderRadius: RatingCapsuleHeight/2}}>
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:67]: Why are you mixing using the stylesheet and defining inline styles? I think that it's only necessary for the `backgroundColour`, but the other style expressions are independent of the rating value.

File: src/components/SearchView.tsx [hunk 17]
```diff
@@ -0,0 +1,163 @@
...
+    <Text style={{fontSize: 11, color: "white"}}>{ratingValToText(ratingValue)}</Text>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:70]: Refactor to own file.

File: src/components/SearchView.tsx [hunk 18]
```diff
@@ -0,0 +1,163 @@
...
+
+const BusinessLogo = ({logoURL}: {logoURL: string}) => (
+  <View style={{paddingRight: 10}}>
+    <Image source={require('../../assets/icon.png')} style={{width: 58, height: 58}} resizeMode="contain"/>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:76]: Refactor to own file and use StyleSheet.

File: src/components/SearchView.tsx [hunk 19]
```diff
@@ -0,0 +1,163 @@
...
+      <View style = {{flexDirection: "row", justifyContent: "flex-end", paddingVertical: 2}}>
+        <Text style={styles.ratingText}>FutureProof Rating:</Text>
+        <RatingCapsule ratingValue={business.futureProofRating}/>
+      </View>
+      <View style = {{flexDirection: "row", justifyContent: "flex-end", paddingVertical: 2}}>
+        <Text style={styles.ratingText}>Consumer Rating:</Text>
+        <RatingCapsule ratingValue={business.userRating}/>
+      </View>
+  </View>
+)
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:101]: Refactor to own file and use StyleSheet.

File: src/components/SearchView.tsx [hunk 20]
```diff
@@ -0,0 +1,163 @@
...
+      marginHorizontal: 25,
+      padding: 12,
+      borderRadius: 10,
+      alignItems: "center"
+    }}
+  >
+    <BusinessLogo logoURL={business.logoURL}/>
+    <SearchItemStats business={business}/>
+  </TouchableOpacity>
+);
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:118]: Refactor to own file and use StyleSheet.

File: src/components/SearchView.tsx [hunk 21]
```diff
@@ -0,0 +1,163 @@
...
+  },
+  {
+    id: '58694a0f-3da1-471f-bd96-145571e29d72',
+    businessName: 'Pret A Manger',
+    distance: '1.5',
+    futureProofRating: 85,
+    userRating: 38,
+    logoURL: '../../assets/icon.png'
+  }
+];
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:30]: These'll need to be removed before sprint-1 gets merged, they're obviously ok for now until Aman sets up the resolvers.

File: src/components/SearchView.tsx [hunk 22]
```diff
@@ -0,0 +1,163 @@
...
+const SearchView = () => {
+  const [selectedId, setSelectedId] = useState(null);
+
+  const renderSearchItem = ({ item } : { item : Business}) => (
+    <SearchItem
+      business={item}
+      onPress={() => {
+        setSelectedId(item.id)
+
+      }}
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:129]: The call to setSelectedId does not have to be wrapped in brackets.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Good stuff! Just need a couple of things moved around but it looks great.

File: src/components/RatingsView.tsx [hunk 23]
```diff
@@ -0,0 +1,23 @@
+import RatingView from '@components/RatingView'
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+const RatingsView = ({ futureProofRating, userRating }: { futureProofRating: number; userRating: number} ) => (
+  <View>
+    <Text style={styles.headingText}>{'RATINGS'}</Text>
+    <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
+      <RatingView ratingName="FutureProof" ratingValue={futureProofRating} />
+      <RatingView ratingName="Consumer" ratingValue={futureProofRating} />
```
Comment (bridges-wood) [inline src/components/RatingsView.tsx:10]: `ratingName` prop should be typed with TS.

File: src/components/SearchView.tsx [hunk 24]
```diff
@@ -0,0 +1,47 @@
...
+
+const SearchView = () => {
+  const [selectedId, setSelectedId] = useState(null)
+
+  const renderSearchItem = ({ item } : { item : DisplayableBusiness}) => (
+    <SearchResult
+      business={item}
+      onPress={() => {
+        //setSelectedId(item.id)
+      }}
```
Comment (bridges-wood) [inline src/components/SearchView.tsx:26]: Why is this here?

File: src/components/TitleView.tsx [hunk 25]
```diff
@@ -0,0 +1,26 @@
+import { DisplayableBusiness } from '@typings/types'
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+type ViewProps = Pick<DisplayableBusiness, 'name' | 'profilePicture'>
+  
+const TitleView = ({ name, profilePicture }: ViewProps ) => (
+  <View style={{ alignItems: 'center' }}>
+    {profilePicture ? <Image source={{uri: profilePicture}} style={{ width: 90, height: 90, borderRadius: 10 }} resizeMode="contain" /> : null}
```
Comment (bridges-wood) [inline src/components/TitleView.tsx:9]: Refactor style prop to stylesheet.

File: src/components/search/RatingCapsule.tsx [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+  if (ratingValue > 30) {
+    return 'Bad'
+  }
+  return 'Avoid'
+}
+  
+const RatingCapsuleHeight = 18
+
+const RatingCapsule = ({ratingValue}: {ratingValue: number}) => (
+  <View style={{alignItems: 'center', backgroundColor: ratingValToColour(ratingValue), width: 55, height: RatingCapsuleHeight, borderRadius: RatingCapsuleHeight/2}}>
```
Comment (bridges-wood) [inline src/components/search/RatingCapsule.tsx:39]: Refactor non-conditional styles to stylesheet.

File: src/components/search/RatingCapsule.tsx [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+    <Text style={{fontSize: 11, color: 'white'}}>{ratingValToText(ratingValue)}</Text>
```
Comment (bridges-wood) [inline src/components/search/RatingCapsule.tsx:40]: Refactor style prop to stylesheet.

File: src/components/search/SearchItemStats.tsx [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+// CODE REQUIRED -> Fetch user location and business location and calculate distance between
+function convertLocationToDistance(businessLocation: Location): number {
+  return 1.2
+}
+
+const SearchItemStats = ({business}: {business: DisplayableBusiness}) => (
+  <View style={styles.searchItemStats}>
+    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <View style = {{flexDirection: 'row'}}>
+      <Image source={require('../../../assets/icon_location.png')} style={{width: 20, height: 20}} resizeMode="contain"/>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:15]: This should use the uri property.

File: src/components/search/SearchItemStats.tsx [hunk 29]
```diff
@@ -0,0 +1,50 @@
...
+
+// CODE REQUIRED -> Fetch user location and business location and calculate distance between
+function convertLocationToDistance(businessLocation: Location): number {
+  return 1.2
+}
+
+const SearchItemStats = ({business}: {business: DisplayableBusiness}) => (
+  <View style={styles.searchItemStats}>
+    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <View style = {{flexDirection: 'row'}}>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:14]: Refactor style prop to stylesheet.

File: src/components/search/SearchItemStats.tsx (hunk 28 above)
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:15]: Refactor style prop to stylesheet.

File: src/components/search/SearchItemStats.tsx [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+      <Image source={require('../../../assets/icon_location.png')} style={{width: 20, height: 20}} resizeMode="contain"/>
+      <Text style={styles.subtitleText}>{'0.0' + ' miles'}</Text>
+    </View>
+      
+    <View style = {{flexDirection: 'row', justifyContent: 'flex-end', paddingVertical: 2}}>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:19]: Refactor style prop to stylesheet.

File: src/components/search/SearchItemStats.tsx [hunk 31]
```diff
@@ -0,0 +1,50 @@
...
+function convertLocationToDistance(businessLocation: Location): number {
+  return 1.2
+}
+
+const SearchItemStats = ({business}: {business: DisplayableBusiness}) => (
+  <View style={styles.searchItemStats}>
+    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <View style = {{flexDirection: 'row'}}>
+      <Image source={require('../../../assets/icon_location.png')} style={{width: 20, height: 20}} resizeMode="contain"/>
+      <Text style={styles.subtitleText}>{'0.0' + ' miles'}</Text>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:16]: Use result of distance function.

File: src/components/search/SearchItemStats.tsx [hunk 32]
```diff
@@ -0,0 +1,50 @@
...
+    </View>
+      
+    <View style = {{flexDirection: 'row', justifyContent: 'flex-end', paddingVertical: 2}}>
+      <Text style={styles.ratingText}>FutureProof Rating:</Text>
+      <RatingCapsule ratingValue={business.sustainabilityScore || 0}/>
+    </View>
+    <View style = {{flexDirection: 'row', justifyContent: 'flex-end', paddingVertical: 2}}>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:23]: Refactor style prop to stylesheet.

File: src/components/search/SearchItemStats.tsx [hunk 33]
```diff
@@ -0,0 +1,50 @@
...
+  <View style={styles.searchItemStats}>
+    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <View style = {{flexDirection: 'row'}}>
+      <Image source={require('../../../assets/icon_location.png')} style={{width: 20, height: 20}} resizeMode="contain"/>
+      <Text style={styles.subtitleText}>{'0.0' + ' miles'}</Text>
+    </View>
+      
+    <View style = {{flexDirection: 'row', justifyContent: 'flex-end', paddingVertical: 2}}>
+      <Text style={styles.ratingText}>FutureProof Rating:</Text>
+      <RatingCapsule ratingValue={business.sustainabilityScore || 0}/>
```
Comment (bridges-wood) [inline src/components/search/SearchItemStats.tsx:21]: Default value should be declared in child component.

File: src/components/search/SearchResult.tsx [hunk 34]
```diff
@@ -0,0 +1,33 @@
+import SearchItemStats from '@components/search/SearchItemStats'
+import { DisplayableBusiness } from '@typings/types'
+import React from 'react'
+import { Image, StyleSheet, TouchableOpacity, View } from 'react-native'
+
+const BusinessLogo = ({logoURL}: {logoURL: string}) => (
+  <View style={{paddingRight: 10}}>
+    <Image source={{uri: logoURL}} style={{width: 58, height: 58}} resizeMode="contain"/>
+  </View>
```
Comment (bridges-wood) [inline src/components/search/SearchResult.tsx:9]: Refactor styles prop to stylesheet.

File: src/components/search/SearchItemStats.tsx (hunk 31 above)
Comment (Matthew5002) [reply src/components/search/SearchItemStats.tsx]: I've removed the distance function as this is something we'll need to review at a later date.

File: src/components/DescriptionView.tsx [hunk 35]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+const DescriptionView = ({ descriptionText }: { descriptionText: string} ) => (
```
[the remark this answers is not in the document]
Comment (Matthew5002) [reply src/components/DescriptionView.tsx:4]: Other than handling conditionals I don't understand why this needs to be changed?

File: src/components/SearchView.tsx (hunk 24 above)
Comment (Matthew5002) [reply src/components/SearchView.tsx]: I've uncommented this. This onPress function can be used to respond to when the user taps a cell/business in the search view

File: src/components/ImagesCarousel.tsx [hunk 36]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+const ImagesCarousel = () => (
+  <View>
+    <Text style={styles.headingText}>{'DESCRIPTION'}</Text>
+    <Image source={require('../../assets/Sample_Business_Image.png')} style={{ width: 250, height: 300, borderRadius: 10, alignSelf: 'center' }} resizeMode="contain" />
```
[the remark this answers is not in the document]
Comment (Matthew5002) [reply src/components/ImagesCarousel.tsx]: This is just a placeholder/prototype for now. Since this feature has not been implemented server-side and was not due to be implemented in sprint 1 I am leaving this for now.

File: src/components/ImagesCarousel.tsx [hunk 37]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+const ImagesCarousel = () => (
+  <View>
+    <Text style={styles.headingText}>{'DESCRIPTION'}</Text>
```
[the remark this answers is not in the document]
Comment (Matthew5002) [reply src/components/ImagesCarousel.tsx]: This will be completed during a later sprint when this functionality has been implemented server-side

File: src/App.tsx (hunk 1 above)
Comment (Matthew5002) [reply src/App.tsx]: I was doing this to make it easy to switch between views for testing and easy to add each view once we implement navigation.

File: src/components/BusinessView.tsx (hunk 6 above)
Comment (Matthew5002) [reply src/components/BusinessView.tsx]: This is meant to be done in sprint 2, I was getting ahead a little here.

File: src/components/RatingView.tsx [hunk 38]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+const RatingView = ({ ratingName, ratingValue }: { ratingName: string; ratingValue: number} ) => (
```
[the remark this answers is not in the document]
Comment (Matthew5002) [reply src/components/RatingView.tsx:4]: The type could be different though? A sustainability score or a consumer score. The parent RatingsView uses the typings from DisplayableBusiness so I think this is fine this way?

File: src/components/DescriptionView.tsx [hunk 39]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import ParagraphIcon from '../../assets/icon_paragraph.png'
+import { DisplayableBusiness } from '@typings/types'
+
+type ViewProps = Pick<DisplayableBusiness, 'profileText'>
+
+const DescriptionView = ({ profileText }: ViewProps) => (
+  <View>
+    <Text style={styles.headingText}>{'DESCRIPTION'}</Text>
```
Comment (bridges-wood) [inline src/components/DescriptionView.tsx:10]: This doesn't have to be wrapped in quotes and brackets.

File: src/components/RatingView.tsx [hunk 40]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+const RatingView = ({ ratingName, ratingValue = 0 }: { ratingName: string; ratingValue?: number | null} ) => (
+  <View style={styles.itemBackgroundStyle}>
+    <View style={styles.ratingContainerStyle}>
+      <Text>{ratingName + ' Rating'}</Text>
```
Comment (bridges-wood) [inline src/components/RatingView.tsx:7]: Use template string here.

File: src/components/RatingView.tsx [hunk 41]
```diff
@@ -0,0 +1,28 @@
...
+      <Text>{ratingValue == null ? 0 : ratingValue}</Text>
+      <Text style={styles.subtitleText}>{'Tap for Details'}</Text>
```
Comment (bridges-wood) [inline src/components/RatingView.tsx:9]: This doesn't have to be wrapped in quotes and brackets.

File: src/components/RatingsView.tsx [hunk 42]
```diff
@@ -0,0 +1,30 @@
+import RatingView from '@components/RatingView'
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import { DisplayableBusiness } from '@typings/types'
+
+type ViewProps = Pick<DisplayableBusiness, 'customerScore' | 'sustainabilityScore'>
+
+const RatingsView = ({ customerScore, sustainabilityScore}: ViewProps ) => (
+  <View>
+    <Text style={styles.headingText}>{'RATINGS'}</Text>
```
Comment (bridges-wood) [inline src/components/RatingsView.tsx:10]: This doesn't have to be wrapped in quotes and brackets.

File: src/components/TitleView.tsx [hunk 43]
```diff
@@ -0,0 +1,34 @@
...
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+type ViewProps = Pick<DisplayableBusiness, 'name' | 'profilePicture'>
+  
+const TitleView = ({ name, profilePicture }: ViewProps ) => (
+  <View style={styles.titleViewStyle}>
+    {profilePicture ? <Image source={{uri: profilePicture}} style={styles.profilePictureStyle} resizeMode="contain" /> : null}
+    <Text style={styles.titleText}>{name}</Text>
+    <Text style={styles.subtitleText}>{'Address (To be implemented)'}</Text>
```
Comment (bridges-wood) [inline src/components/TitleView.tsx:11]: This doesn't have to be wrapped in quotes and brackets.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Nearly perfect, just a couple of tiny changes.
